tools/Environments/DevHome.Environments/Views/LandingPage.xaml

```diff
@@ -159,6 +159,42 @@
                 </Grid>
             </DataTemplate>
 
+            <!-- Per Provider Card Template -->
+            <DataTemplate x:Key="PerProviderComputeSystemTemplate" x:DataType="vm:PerProviderViewModel">
+                <Grid
+                    Visibility="{x:Bind IsVisible, Mode=OneWay}">

```

Comment:
Hey looks like the headings are still visible:

https://github.com/microsoft/devhome/assets/105318831/03e3a6ee-b15f-4e78-8c78-e57cffe085e5

I think we need to also add the 

```C#
_mainWindow.DispatcherQueue.EnqueueAsync(() =>
{
            IsVisible = ComputeSystemAdvancedView.Count > 0;
});
```

code you have in the search handler, in the provider filter handler as well, but looks like There is some extra padding that gets left over at least from my testing.

Heres what it looks like after I add it in:


https://github.com/microsoft/devhome/assets/105318831/ed741f6d-2899-47b1-aabd-28938650404d

Reply:
Thanks for testing this out with three extensions. The gap really needs to be fixed. And I could have sworn the headings missing was fixed in my original PR. I remember testing it. Good catch. 
Fixed.